### DISCUSSION TITLE
Tell me your favorite anime songs!!!!!

### DISCUSSION THREAD

**Discussion by yusei-mi** [NAIST-SE/AttractingContributors#1168] [2024 Attraction Contributions]
What is your favorite anime song?
↓↓↓↓↓Please tell me here!!!↓↓↓↓↓
https://github.com/yusei-mi/AttractingContributors-Anime_Songs.git

**Comment by wangzaixiaoliunai** [comment]
I have created a PR.
Could you please contribute to mine?
https://github.com/wangzaixiaoliunai/AttractingContributors-musical_instruments.git

**Comment by donguri7** [comment]
Could you please contribute to mine?
https://github.com/donguri7/AttractingContributers_Favorite_sports.git

**Comment by Tz-violet** [comment]
hi, I just suggested my favourite one.
can you help me to complete my project: https://github.com/Tz-violet/AttractingContributors.git ?
